crates/sncast/tests/e2e/invoke.rs

```diff
@@ -34,6 +38,80 @@ async fn test_happy_case() {
 
     assert!(matches!(receipt, Invoke(_)));
 }
+#[tokio::test]
+async fn test_tx_reverted() {
```

Comment:
I guess we should rather have an integration test on this change (we already test wait_for_tx transactions in tests/integration/wait_for_tx.rs). 

That way we could keep a distinction between our test files (eg this file should probably test just invoke subcommand, and not waiting for txs). I'm not opposed to have a test that uses --wait in here, but I guess that we shouldn't mix different tests (with different purposes) with each other, it might get too messy and harder to maintain.